scripts/tests/e2e-link-validation.test.mjs

```diff
@@ -77,13 +95,31 @@ describe("Link Validation", () => {
     console.log("\n━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━");
     console.log("Link Validation Summary");
     console.log("━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━");
-    console.log(`Valid:   ${stats.valid.toLocaleString()}`);
-    console.log(`Skipped: ${stats.skipped.toLocaleString()}`);
-    console.log(`Broken:  ${brokenLinks.length.toLocaleString()}`);
-    console.log(`Pages:   ${stats.pages.size.toLocaleString()}`);
+    console.log(`Valid:    ${stats.valid.toLocaleString()}`);
+    console.log(`Skipped:  ${stats.skipped.toLocaleString()}`);
+    console.log(`Warnings: ${warnings.length.toLocaleString()}`);
+    console.log(`Broken:   ${brokenLinks.length.toLocaleString()}`);
+    console.log(`Pages:    ${stats.pages.size.toLocaleString()}`);
```

Comment:
This is useful output, I wonder if there's a way for us to surface it as a Github status summary within the workflow file? [Blog post](https://github.blog/news-insights/product-news/supercharging-github-actions-with-job-summaries/) for reference.